docs/go/quickstart/users-profile-sample-application.md

```diff
@@ -0,0 +1,179 @@
+---
+id: users-profile-sample-application
+title: Users Profile Sample Application
+sidebar_label: Gin-Mongo Sample 2
+tags:
+  - hello-world
+  - go
+  - sdk
+  - tutorial
+---
+
+# Users-Profile
+
+A sample application that get, create, update, and delete the data of a user in the database (MongoDB for this application).
+
+## Prerequisites
+
+1. [Go](https://go.dev/doc/install) 1.16 or later
+2. [Docker](https://docs.docker.com/engine/install/) for running Keploy server
+3. [Thunder Client](https://marketplace.visualstudio.com/items?itemName=rangav.vscode-thunder-client) / [Postman Desktop Agent](https://www.postman.com/downloads/postman-agent/) for testing localhost APIs
+4. Code Editor ([VSCode](https://code.visualstudio.com/download), [Sublime Text](https://www.sublimetext.com/download), etc.)
+
+## Installation
+
+### Start Keploy Server
+
+
+git clone https://github.com/keploy/keploy.git
+cd keploy
+docker-compose up
+
+
+### Start Users-Profile sample application
+
+
+git clone https://github.com/keploy/samples-go
+cd samples-go
+cd users-profile
+go get .
+go run .
```

Comment:
This section need to be block coded too. Otherwise, it will be displayed as a normal text that does not look good.

![Screenshot from 2022-10-04 16-09-50](https://user-images.githubusercontent.com/74777863/193799346-06d5729d-b7a1-4a25-b032-ce0b2103bd1c.png)